database/database.go

```diff
@@ -31,7 +31,7 @@ type Database struct {
 	jsonFilename string
 }
 
-type DatabaseMessage struct {
+type Message struct {
```

Comment:
Agree that renaming the top level struct to `Message` is better because of `database.DatabaseMessage.